src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -73,6 +74,11 @@
 
     // optional arguments
 
+    @Option(name = "-uniqueDocId", usage = "remove duplicated documents with the same doc id when indexing. " +
+      "please note that this option may slow the indexing a lot and if you are sure there is no " +
```

Comment:
I think we've been using `docid` not `docId`, so the option should be `-uniqueDocid`?